setup.py

```diff
@@ -28,13 +28,13 @@
     "configparser",
     "docker",
     "flask",
-    "grpcio<=1.27.2",
+    "grpcio==1.27.2",
```

Comment:
I'm not sure fixing 3rd party library in install_requres is a good practice, because other libraries installed may require a different version. Pinning the dev dependency is fine